### PR TITLE
Updated Snackbar

### LINKED ISSUE #414: Update snackbar
https://material-ui.com/components/snackbars/

### PR DESCRIPTION
**Pull request by dwkaminsky**
## Description
Updated Snackbar to the latest `material-ui`

## Motivation and Context
Closes #414 

## Screenshots (if appropriate):
captured (4)

### CODE REVIEW

File: usage.py [hunk 1]
```diff
@@ -87,6 +87,15 @@
 
         spacer,
 
+        html.Div([
+            html.P([html.Strong('Test for Snackbar')]),
+            html.Button('Open/Close Snackbar', id='snackbar-button'),
+            sd_material_ui.Snackbar(id='snackbar',
+                                    message='You opened the Snackbar!')
+        ]),
```
Comment (coralvanda) [inline usage.py]: Would you also add an example of using the `action` inside the snackbar?

Comment (dwkaminsky) [reply usage.py]: Should I just use an 'X' here?

Comment (dwkaminsky) [reply usage.py]: image

Comment (coralvanda) [reply usage.py]: That looks fine. Does it close the snackbar before the timer otherwise would?

Comment (dwkaminsky) [reply usage.py]: No, and I can't pass in a button component

Comment (dwkaminsky) [reply usage.py]: Description for action: `The action to display. It renders after the message, at the end of the snackbar.`

Comment (coralvanda) [reply usage.py]: Ok. If the action isn't able to perform a function, then let's drop it so there's no confusion.

Comment (dwkaminsky) [reply usage.py]: Ok, good to merge now @coralvanda

**Pull request merged**